code/game/objects/effects/decals/crayon.dm

```diff
@@ -47,3 +47,289 @@ GLOBAL_LIST(gang_tags)
+/obj/effect/decal/cleanable/crayon/symbols/nay
+	icon_state = "nay"
+/obj/effect/decal/cleanable/crayon/symbols/heart
+	icon_state = "heart"
+/obj/effect/decal/cleanable/crayon/symbols/credit
+	icon_state = "credit"
+
+/obj/effect/decal/cleanable/crayon/drawings
+	name = "Drawing"
+	icon_state = "danger"
+
+/obj/effect/decal/cleanable/crayon/drawings/smallbrush
+	icon_state = "smallbrush"
+/obj/effect/decal/cleanable/crayon/drawings/brush
+	icon_state = "brush"
+/obj/effect/decal/cleanable/crayon/drawings/largebrush
+	icon_state = "largebrush"
+/obj/effect/decal/cleanable/crayon/drawings/splatter
+	icon_state = "splatter"
+/obj/effect/decal/cleanable/crayon/drawings/snake
+	icon_state = "snake"
+/obj/effect/decal/cleanable/crayon/drawings/stickman
+	icon_state = "stickman"
+/obj/effect/decal/cleanable/crayon/drawings/carp
+	icon_state = "carp"
+/obj/effect/decal/cleanable/crayon/drawings/ghost
+	icon_state = "ghost"
+/obj/effect/decal/cleanable/crayon/drawings/clown
+	icon_state = "clown"
+/obj/effect/decal/cleanable/crayon/drawings/taser
+	icon_state = "taser"
+/obj/effect/decal/cleanable/crayon/drawings/disk
+	icon_state = "disk"
+/obj/effect/decal/cleanable/crayon/drawings/fireaxe
+	icon_state = "fireaxe"
+/obj/effect/decal/cleanable/crayon/drawings/toolbox
+	icon_state = "toolbox"
+/obj/effect/decal/cleanable/crayon/drawings/corgi
+	icon_state = "corgi"
+/obj/effect/decal/cleanable/crayon/drawings/cat
+	icon_state = "cat"
+/obj/effect/decal/cleanable/crayon/drawings/toilet
+	icon_state = "toilet"
+/obj/effect/decal/cleanable/crayon/drawings/blueprint
+	icon_state = "blueprint"
+/obj/effect/decal/cleanable/crayon/drawings/beepsky
+	icon_state = "beepsky"
+/obj/effect/decal/cleanable/crayon/drawings/scroll
+	icon_state = "scroll"
+/obj/effect/decal/cleanable/crayon/drawings/bottle
+	icon_state = "bottle"
+/obj/effect/decal/cleanable/crayon/drawings/shotgun
+	icon_state = "shotgun"
+
+/obj/effect/decal/cleanable/crayon/oriented
+	name = "Oriented"
+	icon_state = "arrow"
+
+/obj/effect/decal/cleanable/crayon/oriented/arrow
+	icon_state = "arrow"
+/obj/effect/decal/cleanable/crayon/oriented/line
+	icon_state = "line"
+/obj/effect/decal/cleanable/crayon/oriented/thinline
+	icon_state = "thinline"
+/obj/effect/decal/cleanable/crayon/oriented/shortline
+	icon_state = "shortline"
+/obj/effect/decal/cleanable/crayon/oriented/body
+	icon_state = "body"
+/obj/effect/decal/cleanable/crayon/oriented/chevron
+	icon_state = "chevron"
+/obj/effect/decal/cleanable/crayon/oriented/footprint
+	icon_state = "footprint"
+/obj/effect/decal/cleanable/crayon/oriented/clawprint
+	icon_state = "clawprint"
+/obj/effect/decal/cleanable/crayon/oriented/pawprint
+	icon_state = "pawprint"
+
+/obj/effect/decal/cleanable/crayon/letter
+	name = "Letter"
+	icon_state = "a"
+
+/obj/effect/decal/cleanable/crayon/letter/a
+	icon_state = "a"
+/obj/effect/decal/cleanable/crayon/letter/b
+	icon_state = "b"
+/obj/effect/decal/cleanable/crayon/letter/c
+	icon_state = "c"
```

Comment:
lol

Reply:
Runesubtypes have been gutted and replaced with a list of icon states for the graffiti spawner 877b512